lib/sycamore/pyproject.toml

```diff
@@ -13,7 +13,7 @@ packages = [{ include = "sycamore" }]
 "Documentation" = "https://sycamore.readthedocs.io"
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.12"
+python = ">=3.9,<3.9.7 || >3.9.7,<3.12"
```

Comment:
Apparently streamlit doesn't work with it. I have no idea why. Since I'm moving the streamlit app to apps/ this will go away and only apply to the streamlit app!